.buildkite/pipelines/pull_request/security_solution.yml

```diff
@@ -5,6 +5,7 @@ steps:
       queue: ci-group-6
     depends_on: build
     timeout_in_minutes: 120
+    parallelism: 4
```

Comment:
It would be nice to update the current Cypress readme to mention that on CI the execution is parallelized, and when is needed to increase the number of chunks.